_includes/head.html

```diff
@@ -17,6 +17,7 @@
 
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="stylesheet" href="{{ '/css/main.css' | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ '/css/syntax-highlight.css' | prepend: site.baseurl }}">
```

Comment:
This doesn't feel optimal in terms of content delivery -- can we stick with the file being compiled into the `main.css`?

Reply:
How? It's served over h2 so there's no difference

Reply:
It's not preloaded, so there is still a difference. But a single CSS file isn't really going to kill our performance. With that said, it's not especially difficult to put it back into the main CSS bundle.

Reply:
Turns out, it was already in the bundle. This was left over from a previous iteration.